pkg/options/options.go

```diff
@@ -9,6 +9,16 @@ import (
 	"github.com/bxcodec/faker/v4/pkg/interfaces"
 )
 
+var (
+	generateUniqueValues = false
+	ignoreInterface      = false
+	randomStringLen      = 25
+	lang                 = &interfaces.LangENG
+	randomMaxSize        = 100
+	randomMinSize        = 0
+	iBoundary            = &interfaces.DefaultIntBoundary
+)
```

Comment:
do we really need to support this? It will be unsafe for concurrent use.

Reply:
We can use the atomic operation to support concurrent use cases if it's needed. But in my opinion, a global setting is always valuable, especially when it already supports in the old version.

Reply:
hmmm, I'm a bit reluctant about this.
But, if we want this feature, let's ensure it's safe for concurrent use. Can we have that?